starters/svelte-kit-scss/src/routes/(examples)/api-example/+page.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+	import Counter from '$lib/components/Counter.svelte';
+	import PageLayout from '$lib/components/PageLayout.svelte';
+</script>
+
+<PageLayout>
+	<span slot="heading">Fetch Data From API</span>
+	<div>
+		Fetch component works
+	</div>
+</PageLayout>
```

Comment:
I'd try to have a header/main/footer on each page (if possible keep the in the layout)
but:
```suggestion
<Header>
	<h1>Fetch Data From API</span>
</Header>
<main>
	<div>
		Fetch component works
	</div>
</main>
```